### PR TITLE
triton: Use DC us-sw-1 as default and updated example

### PR DESCRIPTION
**Pull request by rickard-von-essen**
- Use `us-sw-1` data center as default
- Update triton example and add triton CLI ref.

Closes: #4814

### CODE REVIEW

**Comment by rickard-von-essen** [comment]
FYI @siepkes

@mwhooker It would be great if you could verify that this is enough for you to get started.

**Comment by jen20** [comment]
LGTM @rickard-von-essen. At some point in the near future there will be a pull request to change out the underlying library for Triton such that the SSH Agent can be used for authentication in a similar manner to Terraform.

File: website/source/docs/builders/triton.html.md [hunk 1]
```diff
@@ -130,17 +131,20 @@ cloud:
...
+      "triton_key_material": "/Users/rickard/.ssh/joyent_id_rsa",
+      "source_machine_package": "g4-highcpu-512M",
       "source_machine_image": "70e3ae72-96b6-11e6-9056-9737fd4d0764",
+      "source_machine_networks": [
+        "1fc62c97-c1f0-41c6-9ef7-1f7ebe0ff09a",
+        "f7ed95d3-faaf-43ef-9346-15644403b963"
+      ],
       "ssh_username": "root",
-      "ssh_private_key_file": "~/.ssh/id_rsa",
+      "ssh_private_key_file": "/Users/rickard/.ssh/joyent_id_rsa",
```
Comment (jen20) [inline website/source/docs/builders/triton.html.md]: Minor nit: do we want to make the username more generic here? IIRC (I may not), we expand "~" for home directories in paths such as this - same on line 135.

**Comment by mwhooker** [comment]
@jen20 according to https://github.com/hashicorp/packer/issues/4814 the tilde expansion doesn't work. Maybe fixing that is the right solution

File: website/source/docs/builders/triton.html.md (hunk 1 above)
Comment (rickard-von-essen) [inline website/source/docs/builders/triton.html.md]: (Unless someone recently added tilde expansion and I didn't notice) tilde expansion is not supported in packer.

**Comment by siepkes** [comment]
@jen20 @mwhooker My tests also showed that using a tilde (~) doesn't work.

On a somewhat unrelated note: I'm on the road for a couple of days so I might be less responsive the next couple of days.

**Comment by mwhooker** [comment]
@rickard-von-essen waiting on these triton PRs until I can get access to a triton cloud account which should be in a couple weeks

**Comment by jen20** [comment]
@mwhooker: the credentials for the HashiCorp Triton account are in
1Password - these are what Terraform use. I have a PR to follow later today
which will replace the underlying Triton client library and fix up many of
the issues discussed over the last few days (including documentation).

-- 
James

**Comment by mwhooker** [comment]
:+1:

**Comment by jen20** [comment]
@rickard-von-essen I'm just looking at reconciling this with #4838 - what was the reasoning for changing the default datacenter?

**Pull request closed without merging**